megamek/src/megamek/common/IBomber.java

```diff
@@ -20,48 +20,79 @@
 
 import java.util.Arrays;
 import java.util.List;
+import java.util.stream.IntStream;
 
 import megamek.common.options.OptionsConstants;
 
 /**
  * Common interface for all entities capable of carrying bombs and making bomb attacks, includig Aero,
  * LandAirMech, and VTOL.
- * 
+ *
  * @author Neoancient
  */
 public interface IBomber {
-    
+
     String SPACE_BOMB_ATTACK = "SpaceBombAttack";
     String DIVE_BOMB_ATTACK = "DiveBombAttack";
     String ALT_BOMB_ATTACK = "AltBombAttack";
 
+    void setUsedInternalBombs(int b);
```

Comment:
I'd love to have a javadoc comment on methods in an interface (as comments are visible in the implementors by hovering) unless it's totally obvious what a method does from the name.

Reply:
Updated.